db/seeds.rb

```diff
@@ -54,6 +153,22 @@
                    price: 150,
                    brand: 'Thirty-Two',
                    image: File.open('app/assets/images/boot2.jpg')
+                 },
+                 { name: 'TRIAD BOOTS',
+                   description: 'The Triad strikes a chord with freestyle riders as the go to performance snowboard boot built to last. Featuring the Tongue Tied Boa
+                                 closure system for next level heel hold and maximum control on a traditional lace snowboard boot. Blending top technologies such as
+                                 the all new FlexLight outsole for better cushioning and custom fitting Intuition Mobile Foam Liner, these lightweight kicks prove
+                                 durable and supportive, park lap after park lap.',
+                   price: 230,
+                   brand: 'Ride',
+                   image: File.open('app/assets/images/triad_boots.png')
+                 },
+                 { name: 'HASHTAG BOOT',
+                   description: 'K2’s Fast-In technology keeps things simple and quick so more time is spent actually snowboarding rather than fighting with a boot
+                                 that would rather stay home.',
+                   price: 170,
+                   brand: 'K2',
+                   image: File.open('app/assets/images/hashtag.png')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.